src/main/java/com/money_account_service/controllers/AuthorizationController.java

```diff
@@ -9,7 +9,11 @@ public abstract class AuthorizationController {
 
     private final AuthorizationService authorizationService;
 
-    public UserModel authorize(String token) {
+    public Long authorize(String token) {
         return authorizationService.authorize(token);
     }
+
+    public UserModel authorizeWithUserModel(String token) {
```

Comment:
I'm not sure we should expose this here. Isn't it going to be used just in one endpoint?

Reply:
yes, but it is needed for account creation

Reply:
I'm not sure I would expose this as part of the controller. We could just leave it in the `AuthorizationService`, because as you noticed - it's used only in one place.

Also, I'm thinking about a slight name change of the functions:
* `authorizeUser` - this retruns the `User` class
* `authorizeAccount` - this returns the `Account` class

And btw. using  the **model** suffix in `UserModel` seems unnatural to me. At least in a situation, where we won't use it for other model classes.